.github/ISSUE_TEMPLATE/--weekly-check-in.md

```diff
@@ -40,6 +40,10 @@ Thanks so much `xyz` <!-- Replace `xyz` with the contributors who worked on the
 ## Next Check-in
 If you would like to open the next check-in..Leave a comment below. We are happy to help if its your first time
 
+### Check-in Ideas
```

Comment:
Just noticed this. I'm not sure if you were supposed to add a new header. I think the issue just asked for the text to be added to the already existing `Next Check-in` header.

But then again, the way you've done it might actually be better/preferable. Let's let @cesswairimu decide 😅 .

Reply:
okay , i will do the changes if required.

Reply:
Thanks  a lot you both :heart: I think we could leave it in the current header for the time being. This is looking good :tada: :tada: